### ISSUE TITLE
Parts of the documentation are broken - jQuery seems to be missing

### ISSUE THREAD

**Issue by wjandrea** [ipython/ipython#14213]
Example page: https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.formatters.html

Instead of the diagram, I'm seeing plain text.
![screenshot](https://github.com/ipython/ipython/assets/22385371/261a00b8-8b8d-48e4-9bab-b4afde72a96f)

I tried to click the "v:stable" button to see if previous versions are working, but the button doesn't do anything.

I'm not a web dev, but I found this in the browser console and I bet that's the problem:
```none
15:25:34.110 IPython.core.formatters.html:973 Uncaught ReferenceError: jQuery is not defined
    at IPython.core.formatters.html:973:7
(anonymous) @ IPython.core.formatters.html:973
```

**Comment by wjandrea** [comment]
I just figured out how to manually edit the URL to go to a different version, and I can confirm the last version (8.15.0) is not affected.
![screenshot](https://github.com/ipython/ipython/assets/22385371/73495492-e9b3-470d-9922-ed058864e417)

**Comment by wjandrea** [comment]
Search is also broken. It works on 8.15.0.

**Comment by Carreau** [comment]
That might be a readthedocs things as I don't think we inject this ourselves.

**Comment by wjandrea** [comment]
@Carreau It looks like Jupyter had the same problem and fixed it by changing their readthedocs config: https://github.com/jupyter/jupyter/commit/f69a14d716f34b5258c768203308ea65cc60df01

I found a readthedocs issue about this: https://github.com/readthedocs/readthedocs.org/issues/8800
"the new builders don't include several pre-installed packages like the older ones"

I'm not sure if that'll fix the search and version selector, but it looks like it'll fix the graph at least.

**Comment by wjandrea** [comment]
OK, search seems to be a separate issue. See #14264

**Comment by astrojuanlu** [comment]
This issue was addressed in #14272 I believe.

For search, see #14346
